pkgs/development/python-modules/pyrodigal/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, cython
+, archspec
+}:
+
+buildPythonPackage rec {
+  pname = "pyrodigal";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    pname = "pyrodigal";
+    inherit version;
+    hash = "sha256-1RZXeqb4F3ryDRdsB1e7zhc2Ha+1z/wosqMs/xJWwyU=";
+  };
+
+  propagatedBuildInputs = [ archspec ];
+
+  buildInputs = [ cython ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "pyrodigal" ];
+
+  meta = {
+    description = "Cython bindings and Python interface to Prodigal, an ORF finder for genomes and metagenomes.";
```

Comment:
Nit: Per [this part](https://nixos.org/manual/nixpkgs/stable/#var-meta-description) of the documentation, please remove the period at the end of the description.